examples/gitops.yaml

```diff
@@ -1,9 +1,97 @@
-# var file path, you can set it to absolute path or relative path.
-varFile: variables-gitops.yaml # here is a relative path. (defaults is ./variables.yaml)
-# tool file path, you can set it to absolute path or relative path.
-toolFile: tools-gitops.yaml # here is a relative path.
-# state config, backend can be local or s3
-state:
+---
+# core config
+varFile: "" # If not empty, use the specified external variables config file
```

Comment:
Maybe we should tell user that toolFile should be not empty when varFile is not empty. In docs or this yaml file.

Reply:
The logic here is not that `toolFile` must be non-empty if `varFile` is not empty, but that if `toolFile` exists and var is used inside, `varFile` needs to exist. If `toolFile` and `toolConfig` does not exist, it does not matter whether `varFile` exists or not. We can "educate" the user later in one of the documents.

Reply:
> The logic here is not that `toolFile` must be non-empty if `varFile` is not empty, but that if `toolFile` exists and var is used inside, `varFile` needs to exist. If `toolFile` and `toolConfig` does not exist, it does not matter whether `varFile` exists or not. We can "educate" the user later in one of the documents.

Got it, thank you for explanation.